File: SofiForce.Models/Models/EntityModels/SalesOrderLogModel.cs
using System;

namespace Models
{
    public class SalesOrderLogModel
    {
        public long LogId { get; set; }
        public long? SalesId { get; set; }
        public DateTime? LogDate { get; set; }
        public string LogStatment { get; set; }
        public int? PurgeAfter { get; set; }
        public string IP { get; set; }
        public string Device { get; set; }
        public string OS { get; set; }
        public string Agent { get; set; }
        public int CBy { get; set; }
        public DateTime? CDate { get; set; }
    }
}

File: SofiForce.Web/ClientApp/src/app/core/Models/EntityModels/SalesOrderLogModel.ts
import { SalesOrderModel } from "./salesOrderModel";

export interface SalesOrderLogModel {
    logId: number;
    salesId: number | null;
    logDate: Date | null;
    logStatment: string;
    purgeAfter: number | null;
    iP: string;
    device: string;
    oS: string;
    agent: string;
    cBy: number;
    cDate: Date | null;
 
}